### PR TITLE
feat: Add border radius to post highlighting border

### PR DESCRIPTION
**Pull request by RiteshK-611**
### Describe what this PR does
- Added border radius to the post highlighting border
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/styles/utilities.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     &:focus-visible {
       box-shadow: 0 0 0 0.125rem var(--theme-focus);
+      border-radius: 15px;
```
Comment (sshanzel) [inline packages/shared/src/styles/utilities.css:7]: Just converting from `px` to `rem` 🥳 

```suggestion
      border-radius: 0.9375rem;
```

**Review by sshanzel — CHANGES_REQUESTED** [review]
Wow that was a quick one! ⚡ 

Just convert the value to using `rem` and we should be good to go :shipit:

File: packages/shared/src/styles/utilities.css (hunk 1 above)
Comment (RiteshK-611) [reply packages/shared/src/styles/utilities.css]: Done! 👍

**Review by sshanzel — APPROVED** [review]
Thank you so much for the swift work! Raising issues is already a great help, fixing the concern yourself is another level 🙏

**Review by rebelchris — APPROVED** [review]
Thanks for this contribution 🙏

**Pull request merged**